domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientTest.java

```diff
@@ -130,6 +126,35 @@ public void configFileTenant() throws Exception {
     assertThat(config.getInteger("connectionReleaseDelay"), is(30000));
   }
 
+  @Test
+  public void testPgConnectOptionsEmpty() {
+    JsonObject conf = new JsonObject();
+    PgConnectOptions options = PostgresClient.createPgConnectOptions(conf);
+    Assert.assertEquals("localhost", options.getHost());
+    Assert.assertEquals(5432, options.getPort());
+    Assert.assertEquals("user", options.getUser());
+    Assert.assertEquals("pass", options.getPassword());
+    Assert.assertEquals("db", options.getDatabase());
```

Comment:
The same assert method should be used throughout a file.
Can you replace `Assert.assertEquals(expected, actual)` by `assertThat(actual, is(expected))` in testPgConnectOptionsEmpty and testPgConnectOptionsFull?

Reply:
Sure